src/App.jsx

```diff
@@ -1,98 +1,13 @@
 /* eslint-disable max-len */
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MovieList } from './components/MovieList/MovieList';
```

Comment:
The imported component 'MovieList' should be 'MoviesList' according to the task description.